### PR TITLE
Update CI scripts

### LINKED ISSUE #15: Broken tests for `wp-cli/embed-command`
Test output from https://travis-ci.org/wp-cli/automated-tests/jobs/335081139:
```
Testing package wp-cli/embed-command...
OS:	Linux 4.14.12-041412-generic #201801051649 SMP Fri Jan 5 16:50:54 UTC 2018 x86_64
Shell:	
PHP binary:	/home/travis/.phpenv/versions/5.6.32/bin/php
PHP version:	5.6.32
php.ini used:	/home/travis/.phpenv/versions/5.6.32/etc/php.ini
WP-CLI root dir:	phar://wp-cli.phar/vendor/wp-cli/wp-cli
WP-CLI vendor dir:	phar://wp-cli.phar/vendor
WP_CLI phar path:	/home/travis/build/wp-cli/automated-tests
WP-CLI packages dir:	/tmp/wp-cli-home/.wp-cli/packages/
WP-CLI global config:	
WP-CLI project config:	
WP-CLI version:	1.2.0-alpha-98b2f93
WordPress 4.9.2
...................................................................... 70
...............................................F--.................... 140
..............F--..................................................... 210
...........................F.......................................... 280
...................................................................... 350
..........................
(::) failed steps (::)
01. $ wp embed fetch https://asciinema.org/a/140798
    <a href="https://asciinema.org/a/140798">https://asciinema.org/a/140798</a>
    
    cwd: /tmp/wp-cli-test-run-fetch.feature.91-5a704f09ae4628.71543217/
    run time: 2.3201439380646
    exit status: 0
    In step `Then the return code should be 1'.                                                   # vendor/wp-cli/wp-cli/features/steps/then.php:7
    From scenario `Get HTML embed code for a given URL that requires discovery and is sanitized'. # vendor/wp-cli/embed-command/features/fetch.feature:91
    Of feature `Manage oEmbed cache.'.                                                            # vendor/wp-cli/embed-command/features/fetch.feature
02. $ wp embed fetch https://foo.example.com
    <a href="https://foo.example.com">https://foo.example.com</a>
    
    cwd: /tmp/wp-cli-test-run-fetch.feature.189-5a704f23c627e5.37435789/
    run time: 0.56786799430847
    exit status: 0
    In step `Then the return code should be 1'.                                                   # vendor/wp-cli/wp-cli/features/steps/then.php:7
    From scenario `Doesn't make unknown URLs clickable'.                                          # vendor/wp-cli/embed-command/features/fetch.feature:189
    Of feature `Manage oEmbed cache.'.                                                            # vendor/wp-cli/embed-command/features/fetch.feature
03. $ wp embed fetch http://video.google.com/videoplay?docid=123456789 --post-id=3
    <a href="http://video.google.com/videoplay?docid=123456789">http://video.google.com/videoplay?docid=123456789</a>
    
    cwd: /tmp/wp-cli-test-run-fetch.feature.403-5a704f81ed8048.03967640/
    run time: 0.67743492126465
    exit status: 0
    In step `And STDOUT should contain:'.                                                         # vendor/wp-cli/wp-cli/features/steps/then.php:15
    From scenario `Invoke built-in Google Video handler'.                                         # vendor/wp-cli/embed-command/features/fetch.feature:403
    Of feature `Manage oEmbed cache.'.                                                            # vendor/wp-cli/embed-command/features/fetch.feature
34 scenarios (31 passed, 3 failed)
376 steps (369 passed, 4 skipped, 3 failed)
3m16.647s
```

According to the above output, we're using WordPress version 4.9.2, but a failure happens in a test that is tagged with `@less-than-wp-4.9 @require-wp-4.4`: https://github.com/wp-cli/embed-command/blob/master/features/fetch.feature#L90

### PR DESCRIPTION
**Pull request by schlessera**
Fixes #15

### CODE REVIEW

File: ci/prepare.sh [hunk 1]
```diff
@@ -34,4 +38,3 @@ echo $CLI_VERSION > PHAR_BUILD_VERSION
 
 mysql -e 'CREATE DATABASE wp_cli_test;' -uroot
 mysql -e 'GRANT ALL PRIVILEGES ON wp_cli_test.* TO "wp_cli_test"@"localhost" IDENTIFIED BY "password1"' -uroot
-mysql -e 'GRANT ALL PRIVILEGES ON wp_cli_test_scaffold.* TO "wp_cli_test"@"localhost" IDENTIFIED BY "password1"' -uroot
```
Comment (gitlost) [inline ci/prepare.sh]: Not sure but I think we might need to keep the `GRANT ALL PRIVILEGES ON wp_cli_test_scaffold.*`line here.

Comment (schlessera) [reply ci/prepare.sh]: Good catch, re-added that one.

**Comment by schlessera** [comment]
Oh, wait, another thing to fix...

**Comment by schlessera** [comment]
I just noticed that we never merged the changes from the automated test suite back into the framework files. I hate the fact that the testing setup is not a stand-alone repo...

**Comment by gitlost** [comment]
Okay maybe just update `behat-tags.php` for now?

**Comment by schlessera** [comment]
Yes. I created an issue so we don't forget: https://github.com/wp-cli/wp-cli/issues/4651
But that ain't a priority for now.

**Comment by schlessera** [comment]
I'm force-merging this, as it will probably still contain errors from other packages anyway.

**Pull request merged**